lib/copy.go

```diff
@@ -23,9 +24,15 @@ import (
 	"github.com/appc/acbuild/Godeps/_workspace/src/github.com/coreos/rkt/pkg/uid"
 )
 
-// Copy will copy the directory/file at from to the path to inside the untarred
-// ACI at a.CurrentACIPath.
-func (a *ACBuild) Copy(from, to string) (err error) {
+// Copy has two modes of operation, selected with the explicitTarget flag.
```

Comment:
OK. Now I'm going to annoy you by suggesting something I should have earlier, but only just occurred to me.

This should be two different functions for the two different codepaths. The `explicitTarget` conditional should be handled in the CLI code instead.